IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,212 @@
+                    out a city or town. You are going to need to prompt the user for information
+                    and be able to save that information in a variable using <Code>Console.WriteLine</Code>,
+                    <Code>Console.ReadLine</Code>, and the basics of <Code>variables</Code>.
+                </Text>
+            </AdventureDialog>
+        </Item>
+        <Item>
+            <AdventureDialog>
+                <Text>
+                    If you haven't explored these concepts yet, why not checkout <b>@VariableBasics.PageRef.Name</b>
+                    and <b>@UserInput.PageRef.Name</b>. Once you've practiced these skills then I am sure you'll be
+                    able to help me! Don't be afraid to take a break and come back to the problem if you get stuck either,
+                    I sure have!
+                </Text>
+            </AdventureDialog>
+        </Item>
+     </Adventure>
+</Section>
+    <Section Title="Learning Objectives">
+    <MudList Clickable="true">
+        <MudListItem Icon="@Icons.Filled.Circle">Practice using <Code>Console.WriteLine</Code> method.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice working with <Code>strings</Code>.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice working with <Code>integers</Code>.</MudListItem>
+        <MudListItem Icon="@Icons.Filled.Circle">Practice changing the color of the terminal.</MudListItem>
+    </MudList>
+</Section>
+
+<Section Title="Where to start?">
+    <Text>
+        When you think of many of your favorite games what comes to mind? Perhaps you think of an astronaut
+        flying through an astreroid field in space like in No Man's Sky. Or maybe you think of a plumber jumping
+        on evil turtles to Store a princess like Mario. Maybe it is even something scarier where you are trapped
+        in a Kid's Pizza store with animatronics trying to come to life and get you!
+    </Text>
+    <Text>
+        There are so many different settings that games or stories might take place but the first thing you should figure out
+        is what kind of "genre" the setting is in. Genre is a term that generally means the particular "style" and
+        "world" of the game or story. By knowing your genre you know what places <i>could</i> be possible and what places 
+        wouldn't be possible. you wouldn't expect to see a space level in a fantasy game like Skyrim!
+    </Text>
+    <Text>
+        To help the user figure out what genre the setting is in, let's give our users a list of options to help them
+        generate their own ideas for a genre. You can use the following list as a base but feel free to add more if you want!
+    </Text>
+    <Note>
+        <b>Extra Challenge!</b><br>
+        Can you figure out how to add bullet points to your list as youll? Hint: You should check out the <b>@AsciiArt.PageRef.Name</b> lesson
```

Comment:
Double check sentence: "Can you figure out how to add bullet points to your list as youll?"